requirements.txt

```diff
@@ -1,5 +1,13 @@
+black==24.4.1
```

Comment:
do we really need black for just a simple script?

Reply:
No, I think it was added as a dependency of another package. Removing it.